filters/types.go

```diff
@@ -20,10 +20,18 @@ const (
 	ExistsEndsCI     = "existsEndsCI"
 	ExistsContains   = "existsContains"
 	ExistsContainsCI = "existsContainsCI"
-	ExistsRegexp     = "existsRegexp"
+	ExistsRegexp     = "existsRegexp" // REGEXP_CONTAINS, not anchored.
```

Comment:
Might be worth adding this comment to case insensitive variant too

Reply:
done